src/stores/gameList.js

```diff
@@ -52,14 +52,16 @@ export const useGameListStore = defineStore('gameList', {
     joinGame(data) {
       const updatedGame = this.openGames.find((game) => game.id === data.gameId);
       if (updatedGame) {
-        updatedGame.numPlayers++;
+        // updatedGame.numPlayers++;
+        updatedGame.numPlayers = Math.min(2, updatedGame.numPlayers + 1)
         updatedGame.status = data.newStatus;
       }
     },
     otherLeftGame(gameId) {
       const updatedGame = this.openGames.find((game) => game.id === gameId);
       if (updatedGame) {
-        updatedGame.numPlayers--;
+        // updatedGame.numPlayers--;
+        updatedGame.numPlayers = Math.min(2, updatedGame.numPlayers - 1)
```

Comment:
Right idea but the problem for leaving games is that the count is sometimes negative. So instead of Math.min with 2, we should use Math.max with 0
```suggestion
        updatedGame.numPlayers = Math.max(0, updatedGame.numPlayers - 1);
```